.github/workflows/kind-e2e.yaml

```
@@ -93,6 +93,10 @@ jobs:
         sudo swapoff -a
         sudo rm -f /swapfile
 
+        # Use tmpfs for etcd storage
```

yeah - otherwise it's not obvious

```suggestion        
        # Use in-memory storage to avoid etcd server timeouts.
        # https://kubernetes.slack.com/archives/CEKK1KTN2/p1615134111016300
        # https://github.com/kubernetes-sigs/kind/issues/845
```